docs/qa/docker.rst

```diff
@@ -0,0 +1,116 @@
+Docker for QA
+=============
+The following describes how to get a prebuilt instance of Normandy for
+testing. This is not suitable for development.
+
+Prerequisites
+-------------
+This guide assumes you have already installed and set up the following:
+
+1. Docker_
+2. `Postgres 9.4`_
+
+.. _Docker: https://docs.docker.com/engine/installation/
+.. _Postgres 9.4: http://www.postgresql.org/
+
+Preparing an environment file
+-----------------------------
+Normandy gets all of its configuration via environment variables. Docker
+has built in support for this pattern. For this set up, we will be using
+and environment file to specifiy this configuration, and having Docker
+read that file.
+
+The file can be named anything, but canonically is named ``.env``. The
```

Comment:
We should probably just have them name it `.env` rather than bother pointing out that the name is optional.